From 0e504f890152ef8c96f4a5913dddd8179dbc1f71 Mon Sep 17 00:00:00 2001
From: "max.ma" <max.ma@starfivetech.com>
Date: Wed, 9 Feb 2022 23:04:46 -0800
Subject: [PATCH 09/12] Revert "RISC-V: Reserve about 5K space in mcontext_t to
 support future ISA expansion."

This reverts commit bcface37cf6df3f3b0800f79e4282a690122b0c8.
---
 sysdeps/unix/sysv/linux/riscv/sys/ucontext.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sysdeps/unix/sysv/linux/riscv/sys/ucontext.h b/sysdeps/unix/sysv/linux/riscv/sys/ucontext.h
index 80caf078fc..cfafa44f99 100644
--- a/sysdeps/unix/sysv/linux/riscv/sys/ucontext.h
+++ b/sysdeps/unix/sysv/linux/riscv/sys/ucontext.h
@@ -82,8 +82,6 @@ typedef struct mcontext_t
   {
     __riscv_mc_gp_state __gregs;
     union  __riscv_mc_fp_state __fpregs;
-    /* 5K + 256 reserved for vector state and future expansion.  */
-    unsigned char __reserved[5376] __attribute__ ((__aligned__ (16)));
   } mcontext_t;
 
 /* Userlevel context.  */
-- 
2.33.1

